DQM/SiStripMonitorCluster/src/SiStripMonitorCluster.cc

```diff
@@ -709,7 +709,7 @@ void SiStripMonitorCluster::createMEs(const edm::EventSetup& es,
             FEDCluster.getParameter<double>("xmax"),
             FEDCluster.getParameter<int32_t>("Nbinsy"),
             FEDCluster.getParameter<double>("ymin"),
-            FEDCluster.getParameter<double>("ymax"));
+            FEDCluster.getParameter<double>("ymax"),"");
```

Comment:
what is the purpose of this additional empty char argument "" ?

Reply:
Hello @fabiocos, adding the empty char we get the errors in the TProfile as the RMS/sqrt(entries in the bin), without the empty char we get just the RMS of the bins.

Reply:
@fioriNTU's comment
`
it makes sense, if we do not add the blank string the errors in the TProfile are computed with the RMS of the bins, adding this instead we get the errors as the RMS/sqrt(entries in the bin) that is what we want to have in the plot since this is the correct error on the average value
`

But no, it makes no sense to me. You ask for a double and pass an empty string, which, returns either NaN or zero. At least pass a zero or a NaN then.

Besides, if this something required to make TProfile work, please put and encode into the code which books this TProfile.

Reply:
Ah, now I see. My mistake - the "" is outside the getParameter parenthesis.